projects/libsodium/secret_key_auth_fuzzer.cc

```diff
@@ -0,0 +1,22 @@
+extern "C" {
```

Comment:
Is this necessary?

Reply:
I assumed it was, but it isn't. libsodium uses the `__cplusplus` macro to conditionally include the relevant `extern C` invocations.  I've recompiled both fuzzers without this and they run as expected. Thanks :)